packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -618,6 +676,44 @@ class _BottomNavigationTile extends StatelessWidget {
 }
 
 
+// If the orientaion is landscape and layout is
+// BottomNavigationBarLandscapeLayout.linear then return a
+// icon-space-label row, where space is 8 pixels. Otherwise return a
+// icon-label column.
+class _Tile extends StatelessWidget {
+ const  _Tile({
+    Key? key,
+    required this.layout,
+    required this.icon,
+    required this.label
+  }) : super(key: key);
+
+  final BottomNavigationBarLandscapeLayout layout;
+  final Widget icon;
+  final Widget label;
+
+  @override
+  Widget build(BuildContext context) {
+    final MediaQueryData data = MediaQuery.of(context);
+    if (data.orientation == Orientation.landscape && layout == BottomNavigationBarLandscapeLayout.linear) {
+      return Align(
+        heightFactor: 1,
+        child: Row(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          mainAxisSize: MainAxisSize.min,
+          children: <Widget>[icon, const SizedBox(width: 8), label],
```

Comment:
Is the 8 pixel gap a Material Design spec?

Reply:
Yes, although it comes from me eye-balling the diagrams, not from an explicitly documented value.